src/proof/lfsc/lfsc_node_converter.h

```diff
@@ -169,9 +172,10 @@ class LfscNodeConverter : public NodeConverter
   std::unordered_set<Node> d_symbols;
   /**
    * Mapping from user symbols to the (list of) symbols with that name. This
-   * is used to resolve symbol overloading, which is forbidden in LFSC.
+   * is used to resolve symbol overloading, which is forbidden in LFSC. We use
+   * Node identifiers, since this map is used for both Node and TypeNode.
    */
-  std::map<std::string, std::vector<Node> > d_userSymbolList;
+  std::map<std::string, std::vector<unsigned long> > d_userSymbolList;
```

Comment:
```suggestion
  std::map<std::string, std::vector<uint64_t>> d_userSymbolList;
```
I think we should use `uint64_t` elsewhere too.

Reply:
Sure, updating `Node::getId` should be done separate from this PR though :)